### PR TITLE
Translate to-ruby-from-php (zh-cn)

### PR DESCRIPTION
**Pull request by iLtc**

### CODE REVIEW

File: zh_cn/documentation/ruby-from-other-languages/to-ruby-from-php/index.md [hunk 1]
```diff
@@ -4,4 +4,34 @@ title: "从 PHP 到 Ruby - Ruby 官方网站"
...
+* Ruby 是强类型语言，你需要使用 `to_s`、`to_i` 等方法使变量在字符串、整数等类型之间转换，而不能依赖编程语言去完成它。
+* 字符串、数字、数组、散列等都是对象。所以应该用 -1.abs 而不是 abs(-1) 。
+* 调用方法时括号是可以省略的，除非需要明确哪些参数对应哪些调用方法时。
+* 取代了命名约定，例如下划线，标准函数库和扩展都被包装成模块和类。
+* 反射是 Ruby 对象与生俱来的能力，你不需要像在 PHP 5 中一样使用 `Reflection` 类。
+* 变量都是被引用的。
+* 沒有抽象类 `abstract` 或接口 `interface`。
+* 散列和数组不能互换使用。
+* 只有 `false` 和 `nil` 是 false：`0`、`array()` 和 `""` 在条件判断时均为真。
+* 几乎所有东西都是调用方法，甚至是 `raise` (PHP的 `throw`)。
```
Comment (JuanitoFatas) [inline zh_cn/documentation/ruby-from-other-languages/to-ruby-from-php/index.md]: `PHP的` => `PHP 的`.

File: zh_cn/documentation/ruby-from-other-languages/to-ruby-from-php/index.md [hunk 2]
```diff
@@ -4,4 +4,34 @@ title: "从 PHP 到 Ruby - Ruby 官方网站"
...
+
+* Ruby 是强类型语言，你需要使用 `to_s`、`to_i` 等方法使变量在字符串、整数等类型之间转换，而不能依赖编程语言去完成它。
+* 字符串、数字、数组、散列等都是对象。所以应该用 -1.abs 而不是 abs(-1) 。
+* 调用方法时括号是可以省略的，除非需要明确哪些参数对应哪些调用方法时。
+* 取代了命名约定，例如下划线，标准函数库和扩展都被包装成模块和类。
+* 反射是 Ruby 对象与生俱来的能力，你不需要像在 PHP 5 中一样使用 `Reflection` 类。
+* 变量都是被引用的。
+* 沒有抽象类 `abstract` 或接口 `interface`。
+* 散列和数组不能互换使用。
+* 只有 `false` 和 `nil` 是 false：`0`、`array()` 和 `""` 在条件判断时均为真。
```
Comment (JuanitoFatas) [inline zh_cn/documentation/ruby-from-other-languages/to-ruby-from-php/index.md]: `是 false` => `为假`.

**Comment by JuanitoFatas** [comment]
Other than my comments, looks good to me.

**Comment by iLtc** [comment]
I've fixed them.

**Comment by JuanitoFatas** [comment]
Thanks! :+1:

**Comment by JuanitoFatas** [comment]
Deployed: https://www.ruby-lang.org/zh_cn/documentation/ruby-from-other-languages/to-ruby-from-php/

**Pull request merged**